common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -469,8 +469,12 @@ protected boolean idTokenIsNull(@NonNull final ICacheRecord cacheRecord,
         return null == idTokenRecord;
     }
 
-    protected Set<String> addDefaultScopes(@NonNull final TokenCommandParameters commandParameters) {
+    protected Set<String> addDefaultAndOidcScopes(@NonNull final TokenCommandParameters commandParameters) {
         final Set<String> requestScopes = commandParameters.getScopes();
+
+        if (commandParameters.getOidcScopes() != null && !commandParameters.getOidcScopes().isEmpty()) {
```

Comment:
TokenCommandParameters::getScopes() can pretty explicitly return null.  Should we handle that?